auth/client/client_test.go

```diff
@@ -32,14 +36,21 @@ var _ = Describe("Client", func() {
 	var ctx context.Context
 
 	BeforeEach(func() {
+		serverSessionTokenProviderController = gomock.NewController(GinkgoT())
+		serverSessionTokenProvider = authTest.NewMockServerSessionTokenProvider(serverSessionTokenProviderController)
 		serverTokenSecret = authTest.NewServiceSecret()
 		serverTokenTimeout = testHttp.NewTimeout()
 		name = test.RandomStringFromRangeAndCharset(4, 16, test.CharsetAlphaNumeric)
 		logger = logTest.NewLogger()
 		Expect(logger).ToNot(BeNil())
 		serverToken = authTest.NewSessionToken()
+		serverSessionTokenProvider.EXPECT().ServerSessionToken().Return(serverToken, nil).AnyTimes()
```

Comment:
just a query `.EXPECT()` ?? I see its generated I just didn't expect (haha) that casing

Reply:
Yea, this is how https://github.com/golang/mock generates.